components/Contact.tsx

```diff
@@ -1,13 +1,25 @@
-/* eslint-disable react/no-unescaped-entities */
-/* eslint-disable @next/next/no-img-element */
 import React from "react";
 import Image from "next/image";
 import Link from "next/link";
+import { useForm, ValidationError } from "@formspree/react";
```

Comment:
I believe this is not related to this or, please split each feature in a separate PR